### PR TITLE
Documentation updated!

### PR DESCRIPTION
**Pull request by cosmologist10**

### CODE REVIEW

**Review by jainmickey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,16 +14,58 @@ Source code for Foss Events India Website (https://beta.fossevents.in)
...
+Install the following for requirements/development.txt installation:
+```
+sudo dnf  install libxml2-dev libxslt1-dev python-dev  libpq-dev
+pip install rcssmin --install-option="--without-c-extensions"
+pip install rjsmin --install-option="--without-c-extensions"
+pip install django-compressor --upgrade
 pip install -r requirements/development.txt
+```
+
+For migration, we need to alter some trust auth in pg_hba.conf file:
```
Comment (aniketmaithani) [inline README.md:59]: is this necessary ??? @cosmologist10

**Comment by cosmologist10** [comment]
*@aniketmaithani while doing setup, migration gets possible only after
changing the auth in pg_hba.conf file. That's why i thought it is
necessary.*

File: README.md (hunk 1 above)
Comment (theskumar) [reply README.md:59]: this seems to be fine as far as it's a local development machine.

Comment (aniketmaithani) [reply README.md:59]: Okay @theskumar . Never encountered this though during dev.

Comment (bhanuvrat) [reply README.md:59]: @aniketmaithani this is needed only the first time you setup postgresql on your machine.

**Pull request merged**